pkg/backporter/backporter.go

```diff
@@ -278,94 +304,218 @@ func GetBugHandler(client bugzilla.Client) HandlerFuncWithErrorReturn {
 	}
 }
 
+func getClonesTemplateData(bugID int, w http.ResponseWriter, client bugzilla.Client, allTargetVersions sets.String) (*ClonesTemplateData, error) {
+	bug, err := client.GetBug(bugID)
+	if err != nil {
+		w.WriteHeader(http.StatusNotFound)
+		wpErr := writePage(w, "Not Found", errorTemplate, fmt.Sprintf("Bug#%d not found", bugID))
+		if wpErr != nil {
+			http.Error(w, "Error building page.", http.StatusInternalServerError)
+			return nil, wpErr
```

Comment:
I think we want to get both errors here, not only the `writePage` one. This is usually done via `utilerrors`:
```
import utilerrors "k8s.io/apimachinery/pkg/util/errors"
return utilerrors.NewAggregate([]error{err, writePage(w, "Not Found", errorTemplate, fmt.Sprintf("Bug#%d not found", bugID))})
```

(it also filters out nil errors so this will work fine even when `writePage` doesn't return an error)

Reply:
(The comment applies for all the occurences of handling `writeError` after handling other error)

Reply:
I will use utilerrors, but I dont think it would be ok to put everything in a single statement like - 
return utilerrors.NewAggregate([]error{err, writePage(w, "Not Found", errorTemplate, fmt.Sprintf("Bug#%d not found", bugID))})
since I would want to return an error message back.
Or is that not necessary?

Reply:
The `utilerrors.NewAggregate` returns an error which has a message that contains all non-nil errors or what do you mean?

Reply:
What I meant was that I wouldn't be able to write the return errors as a single line(like the example you have given), since in the case of an error occurring I would have to write an error message to the body of the response.